TypeCobol.Test/Misc/LanguageLevel/tcCall.rdzMix.txt

```diff
@@ -0,0 +1,20 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. tcCall.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 var-program-name PIC X(8) VALUE 'somepgm'.
+       01 var-input PIC X.
+       01 var-inout PIC X.
+       01 var-output PIC X.
+       PROCEDURE DIVISION.
+      *OK
+           CALL var-program-name USING var-input var-inout var-output.
+      *KO procedure-style CALL
+Line 13[34,38] <48, Error, Syntax> - TypeCobol-only feature: 'INPUT' keyword is not supported in a Cobol CALL.
+Line 13[12,50] <30, Error, Semantics> - Semantic error: Function not found 'var-program-name' input(Alphanumeric, ?, Alphanumeric) output(Alphanumeric)
+           CALL var-program-name INPUT var-input 
+Line 14[34,39] <30, Error, Semantics> - Semantic error: Symbol IN-OUT is not referenced
+                                 IN-OUT var-inout
```

Comment:
IN-OUT is also a TypeCobol keyword but doesn't get the same message as the other INPUT/OUTPUT labels "TypeCobol-only feature: ..." ?

Reply:
The fact that `IN-OUT` is a _specific_ TypeCobol keyword is the reason why the message is different. In #1968, we modified the scanner so as to yield a `UserDefinedWord` for `IN-OUT` so here we get the corresponding error: `IN-OUT` is considered as one of the input arguments and as it is not defined, we have a semantic error on the use of an undefined variable.
`INPUT` and `OUTPUT` are Cobol keywords so in this PR we add a special check to create a diagnostic describing the wrong use: `INPUT` and `OUTPUT` are not allowed to be used in a CALL statement.

Reply:
Ok, thanks for the explanation!